shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -132,16 +143,17 @@ case class Arrival(Operator: Option[Operator],
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  val excludeTransferPax: Option[Int] => Option[Int] = maybePax =>
-    maybePax.map(_ - TranPax.getOrElse(0)) match {
-      case Some(nonNegative) if nonNegative >= 0 => Some(nonNegative)
-      case Some(_) => Some(0)
-      case None => None
+  val excludeTransferPax: Option[Passengers] => Option[Passengers] = maybePax =>
+    maybePax.map { p =>
+      p.actual.map(_ - p.transit.getOrElse(0)) match {
+        case Some(nonNegative) if nonNegative >= 0 => p
+        case Some(_) => p.copy(actual = Some(0), transit = Some(0))
+        case None => p.copy(actual = None)
+      }
     }
 
-
-  val bestPcpPaxEstimate: TotalPaxSource = {
-    val preferredSources: List[(FeedSource, Option[Int] => Option[Int])] = List(
+  val bestPaxEstimate: PaxSource = {
```

Comment:
This function can be simplified now. It doesn't need to call `excludeTransferPax` (which we should remove)